tests/layer_tests/pytorch_tests/test_bernoulli.py

```diff
@@ -68,10 +73,13 @@ class aten_bernoulli(torch.nn.Module):
             def __init__(self, p, seed) -> None:
                 super().__init__()
                 gen = torch.Generator()
-                gen.manual_seed(seed)
                 self.gen = gen
                 self.p = p
+                self.seed = seed
                 self.forward = self.bernoulli_with_p
+            def __call__(self, *args, **kwargs):
+                super().__call__(*args, **kwargs)
+                self.gen.manual_seed(self.seed)
```

Comment:
why do we need it? Why `forward` method is not sufficient? It can be a case that you trace a different function that does not include `torch.bernoulli`